File: TallerElectrodomestico-servidor/src/test/java/com/ceiba/electrodomestico/dominio/modelo/TipoElectrodomestico.java
package com.ceiba.electrodomestico.dominio.modelo;

public class TipoElectrodomestico {

    private long id;
    private String marca;
    private String descripcion;

    public TipoElectrodomestico(long id, String marca, String descripcion) {
        this.id = id;
        this.marca = marca;
        this.descripcion = descripcion;
    }

    public long getId() {
        return id;
    }

    public String getMarca() {
        return marca;
    }

    public String getDescripcion() {
        return descripcion;
    }
}

File: TallerElectrodomestico-cliente/src/app/feacture/components/tipo-electrodomestico/model/TipoElectrodomestico.ts
export class TipoElectrodomestico {
    id: number;
    marca: string;
    descripcion: string;

    constructor(id:number, marca:string,descripcion:string){
        this.id=id;
        this.marca=marca;
        this.descripcion=descripcion;
    }
       
}